lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1447,8 +1449,10 @@ mkApiTransactionFromInfo
     -> TransactionInfo
     -> m (ApiTransaction n)
 mkApiTransactionFromInfo ti (TransactionInfo txid ins outs ws meta depth txtime txmeta) = do
-    apiTx <- liftIO $ mkApiTransaction ti txid (drop2nd <$> ins) outs ws (meta, txtime) txmeta $
-        case meta ^. #status of
+    let deposits = []
+    apiTx <- liftIO $ mkApiTransaction ti txid (drop2nd <$> ins) outs ws (meta, txtime) txmeta
+        deposits
+        $ case meta ^. #status of
```

Comment:
This is no more readable than before. It's actually worse. The 80-column max-length rule should apply, but not be detrimental to readability.  If it goes slightly beyond 80 chars, we can live with it.